docs(flutter): Document Flutter desktop limitations (#10490)
getsentry/sentry-docs: docs/platforms/flutter/features/index.mdx

- Integrations for sqflite, routing and more. For a complete list, see our [integrations list](/platforms/flutter/integrations/).
- [Release Health](/product/releases/health/) tracks crash free users and sessions.
- Limited support for Flutter Web.

**Desktop Limitations:**

> Sentry has limited support for Flutter Desktop (Linx & Windows), but there are limitations compared with mobile platforms:

Reviewer: ```suggestion
Sentry has support for Flutter Desktop (Linux & Windows), but there are limitations compared with mobile platforms:
```

Reviewer 2: we should also clarify what support on macOS looks like since users will expect macOS to fall under the `Desktop` category

> Sentry has limited support for Flutter Desktop (Linx & Windows), but there are limitations compared with mobile platforms:

Reviewer 2: ```suggestion
Sentry supports Flutter on Linux and Windows as well, but there are limitations compared with mobile platforms:
```

Reviewer 2: we should also clarify what support on macOS looks like

Reviewer 2: we should also clarify what support on macOS looks like since users will expect macOS to fall under the `Desktop` category

- No native crash support on Windows
- No support for [Release Health](/product/releases/health/)

> - Device context data is not as rich as on Andorid, iOS & macOS

Reviewer: ```suggestion
- Device context data is not as rich as on Android, iOS & macOS
```

- Offline caching isn't supported for exceptions.
- [Release Health](/product/releases/health/) isn't supported.
- Device context data isn't as rich on the web as it is on mobile.

**Desktop Limitations:**

Sentry supports Flutter on Linux and Windows as well, with the following limitations:

> - No native crash support on Windows
> - No support for [Release Health](/product/releases/health/)
> - Device context data is not as rich as on Android, iOS & macOS

Reviewer 2: ```suggestion
- Native crashes are not supported on Windows.
- [Release Health](/product/releases/health/) isn't supported.
- Device context data isn't as rich on the web as it is on mobile.
```

Reviewer: I think `on the web` should be `on Linux and Windows`

> - Device context data isn't as rich on the web as it is on mobile.

Reviewer: ```suggestion
- Device context data isn't as rich on Linux and Windows as it is on mobile.
```

I think it should be like this

Author: sry 🤦
